atd-vzd/metadata/databases/default/tables/public_atd_txdot_crashes.yaml

```diff
@@ -116,6 +116,233 @@ array_relationships:
         remote_table:
           name: atd_txdot_units
           schema: public
+insert_permissions:
```

Comment:
I went thru all of these columns in the crashes table to make sure none of them are PII or sensitive information we would only want admin level users to insert/edit, but I couldn't find any. So I toggled on all columns for insert/update on Editor permissions, let me know if yall see something that should be blocked for Editors.